language-design.md

```diff
@@ -34,6 +50,8 @@ How might the themes of _Growing a Language_ relate to ideas from the Fowler rea
 
 **Response**
 
+The themes of _Growing a Language_ relate to ideas from the Fowler reading in that _Growing a Language_ seeks to make programming languages be more like natural language, as he pushes for that and urges readers to use natural language more similarly to how programming languages are used. Fowler also makes some suggestions to make DSLs more readable, though he states "Don't try to make the DSL read like natural language". This is interesting because they both recognize that there is issues with the design of programming languages, and yet there is some conflict on the boundaries between natural language and programming languages.
```

Comment:
My takeaway from the end of Growing a Language wasn't that programming languages should be more like natural language, but vice versa! Steele seemed to think that working from a simpler vocabulary that was rigorously defined and expanded over time allowed him to communicate his ideas more clearly.

One other point of comparison I took from the two readings is that Growing a Language thinks that small languages should be designed to, well, grow, while Fowler pushes back on that, stating that we should "guard firmly against" unnecessary expansion. While Steele speaks mainly from the perspective of a general-purpose language, these two ideas for how a language should operate are interesting. Maybe the solution is to design your DSL in a limited fashion, but in a ‘pattern’ as Steele suggests, leaving room for expansion if it make sense?